packages/jest-haste-map/src/types.ts

```diff
@@ -9,6 +9,7 @@ import type {Stats} from 'graceful-fs';
 import type {Config} from '@jest/types';
 import type HasteFS from './HasteFS';
 import type ModuleMap from './ModuleMap';
+import type {IModuleMap} from '.';
```

Comment:
move the type into this file and import in `index` instead? This creates a cyclic dependency (albeit type-only, so not a dealbreaker)

Reply:
I moved all the types into types.ts, including SerializableModuleMap. Let me know if this looks good to you

Reply:
yep! 👍